src/main/java/org/nguh/nguhcraft/data/NguhcraftDataGenerator.kt

```diff
@@ -72,6 +72,11 @@ class NguhcraftBlockTagProvider(
 
         getOrCreateTagBuilder(BlockTags.WOODEN_STAIRS).add(NguhBlocks.TINTED_OAK_STAIRS)
         getOrCreateTagBuilder(BlockTags.WOODEN_FENCES).add(NguhBlocks.TINTED_OAK_FENCE)
+        getOrCreateTagBuilder(BlockTags.LOGS_THAT_BURN)
+            .add(NguhBlocks.TINTED_OAK_LOG)
+            .add(NguhBlocks.TINTED_OAK_WOOD)
+            .add(NguhBlocks.STRIPPED_TINTED_OAK_LOG)
+            .add(NguhBlocks.STRIPPED_TINTED_OAK_WOOD)
```

Comment:
Make sure that the logs also end up in the `AXE_MINEABLE` tag (I think for the planks that happens automatically because they’re added to `PLANKS`; I’m not sure if adding the logs to `LOGS_THAT_BURN` is enough for that or if there’s a separate `LOGS` tag that handles that).